### DISCUSSION TITLE
I am Using Replit I get the error SyntaxError: Cannot use import statement outside a module

### DISCUSSION THREAD

**Discussion by 9-SK** [GrapesJS/grapesjs#4025] [Q&A]
Hi, How do I fix this issue?
What is shows 
/home/runner/Ayyy/index.js:1
import 'grapesjs/dist/css/grapes.min.css'
^^^^^^

SyntaxError: Cannot use import statement outside a module
I used nodejs 12, 17, 6 they all did not work

**Comment by ronaldohoch** [comment]
I think this can help you: https://stackoverflow.com/a/58212338/2245999
